prog/weaveutil/addrs.go

```diff
@@ -27,10 +27,25 @@ func containerAddrs(args []string) error {
 		return err
 	}
 
+	containerArgs := []string{}
+	for _, cid := range args[1:] {
+		if cid == "weave:allids" { // expand to all container IDs
```

Comment:
OK, I see.

I thought about introducing `--all` flag instead of special container id `weave:allids`. However, given that one place does not require `weave:expose`, in the other place we would need to call `weaveutil container-addrs --all weave:expose` which IMO looks a bit confusing.